### PR TITLE
completed html-forms and package-lock.json update - brighitte johnson

### PR DESCRIPTION
**Pull request by brij8**
html-forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+
+    <h1>Coding Form!</h1>
+    <form action=" " method="post">
+      <p>Email <br />
```
Comment (Zhiwutian) [inline html-forms/index.html:10]: this should be a label, and make sure that you avoid using `br` tags. breaking input elements on to separate lines should be done using `divs`

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
nice initial submission! there are some updates. Make sure to avoid using `br` tags and to have a label for each input. Also, make sure the labels have a proper `for` attribute with a matching `id` attribute on the input

**Comment by brij8** [comment]
i think i got it all. i'm guessing the spacing would be adjusted in css? 'cause it's very squish atm
html-forms02

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        <select name="codingcoolbc">
+          <option value="selected">Select an option</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="fun">Because it's fun</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+
+      <div class="favlang">
+        <label for="favlang">Favorite Language (choose one)</label>
```
Comment (Zhiwutian) [inline html-forms/index.html:31]: only thing to update here is to have a label for each radio input. you can wrap them like this if you prefer
```html
<label>
    <input type="radio" name="favoriteLanguage" value="HTML">
    HTML
 </label>

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one tiny change!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**